comp/collector/collector/collectorimpl/collector.go

```diff
@@ -132,6 +139,35 @@ func newCollector(deps dependencies) *collectorImpl {
 	return c
 }
 
+// fillFlare collects all the information related to integrations that need to be added to each flare
+func (c *collectorImpl) fillFlare(fb flaretypes.FlareBuilder) error {
+	scanner := scanner.GetGlobalScanner()
```

Comment:
Why is this part of the collector component? Would it make sense to create a dedicated component for this, since collector does not otherwise interact with sbom?

Reply:
The gollector component is responsible for the checks, and the check is not a component yet.